[2023-05-31T02:05:42.717+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: captura_dados.created_folders_task manual__2023-05-31T02:05:40.992922+00:00 [queued]>
[2023-05-31T02:05:42.730+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: captura_dados.created_folders_task manual__2023-05-31T02:05:40.992922+00:00 [queued]>
[2023-05-31T02:05:42.730+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-05-31T02:05:42.730+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-05-31T02:05:42.730+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-05-31T02:05:42.745+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): created_folders_task> on 2023-05-31 02:05:40.992922+00:00
[2023-05-31T02:05:42.751+0000] {standard_task_runner.py:55} INFO - Started process 1733 to run task
[2023-05-31T02:05:42.755+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'captura_dados', 'created_folders_task', 'manual__2023-05-31T02:05:40.992922+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/captura_dados.py', '--cfg-path', '/tmp/tmpjq0yu0kj']
[2023-05-31T02:05:42.757+0000] {standard_task_runner.py:83} INFO - Job 49: Subtask created_folders_task
[2023-05-31T02:05:42.833+0000] {task_command.py:388} INFO - Running <TaskInstance: captura_dados.created_folders_task manual__2023-05-31T02:05:40.992922+00:00 [running]> on host 5219a5ad576c
[2023-05-31T02:05:42.909+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=test@test.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=captura_dados
AIRFLOW_CTX_TASK_ID=created_folders_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-31T02:05:40.992922+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-31T02:05:40.992922+00:00
[2023-05-31T02:05:42.917+0000] {logging_mixin.py:137} INFO - Pasta data j√° existe!
[2023-05-31T02:05:42.917+0000] {logging_mixin.py:137} INFO - /opt/***/engineer
[2023-05-31T02:05:42.918+0000] {logging_mixin.py:137} INFO - /opt/***/raw
[2023-05-31T02:05:42.918+0000] {logging_mixin.py:137} INFO - /opt/***/refined
[2023-05-31T02:05:42.918+0000] {logging_mixin.py:137} INFO - /opt/***/transient
[2023-05-31T02:05:42.918+0000] {logging_mixin.py:137} INFO - /opt/***/trusted
[2023-05-31T02:05:42.919+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-31T02:05:42.933+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=captura_dados, task_id=created_folders_task, execution_date=20230531T020540, start_date=20230531T020542, end_date=20230531T020542
[2023-05-31T02:05:42.967+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-05-31T02:05:42.989+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
